ui/src/scss/_patterns_tables.scss

```diff
@@ -41,4 +41,18 @@
     padding-top: $spv-inner--medium;
     text-transform: uppercase;
   }
+
+  tr:not(:first-child).indented-border {
```

Comment:
We have indented borders in the machine list (grouping), and for PCI devices I did something similar but called it `truncated-border`, so I've created an issue to make indented borders a generic class.

https://github.com/canonical-web-and-design/maas-ui/issues/2113

Reply:
Thanks for creating that issue.